app/templates/components/station-autocomplete.hbs

```diff
@@ -1,8 +1,5 @@
-<input id='autocomplete-input' list="stations-list" class="form-control" placeholder="Search station…" />
 {{#if stations.isLoaded}}
-<datalist id='stations-list' >
-  {{#each stations.all as |station|}}
-  <option value="{{station.id}}">{{station.name}} ({{station.id}})</option>
-  {{/each}}
-</datalist>
+{{#power-select search=(action "findStation") searchField="name" selected=selected onchange=(action "displayStation") as |station|}}
```

Comment:
Does it still work if you break that line, after `selected`?

It would be clearer IMHO.